doc/extension_developer_guide.md

```diff
@@ -96,3 +96,14 @@ EITFJ works without additional configuration. During development you can however
 * `buildExtension`: Set this to `false` in order to skip building the extension definition before the tests. Use this to speedup tests when the extension definition is not modified.
 * `buildExtensionManager`: Set this to `false` to skip building/installing the extension manager binary. Use this to speedup tests when extension manager is not modified.
 * `extensionManagerVersion`: Version of EM to use during tests. By default EITFJ uses the same version as the version defined in `pom.xml` for `extension-manager-integration-test-java`. Changing this is not recommended.
+
+### Integration Test Base Classes
+
+EITFJ contains two abstract base classes for integration tests that contain basic tests useful for most extensions:
```

Comment:
Do the integration test utilities have a dedicated short tag EITFJ?

File [https://github.com/exasol/extension-manager/blob/main/extension-manager-integration-test-java/error_code_config.yml](error_code_config.yml) says: `EMIT` (which I think could be mixed up with the English word "emit").

Reply:
Replaced EMIT with EITFJ